### DISCUSSION TITLE
Demo Shopify

### DISCUSSION THREAD

**Discussion by isyan** [searchspring/snap#1227] [Q&A] [answered]
Hi!
I've just started working on demo shopify implementation (https://github.com/searchspring-implementations/demo.shopify) for a client. This is also my first time working on snap/preact. I would like to ask if this demo shopify implementation includes the search page filter/facet sidebar?

**Comment by korgon** [comment, marked as answer]
Hello @isyan,

The Shopify demo does indeed include search pagination and faceting. You can see the integration in a mockup here:
https://snapui.searchspring.io/y56s6x/index.html
